src/PerformanceSensitiveAnalyzers/CSharp/Microsoft.CodeAnalysis.CSharp.PerformanceSensitiveAnalyzers.csproj

```diff
@@ -13,4 +13,5 @@
   <ItemGroup>
     <ProjectReference Include="..\Core\Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers.csproj" />
   </ItemGroup>
+  <Import Project="..\..\Utilities\Workspaces\Workspaces.Utilities.projitems" Label="Shared" />
```

Comment:
❗️ This is an analyzer package, not a code fix package. It cannot reference the workspace layer.